code/modules/client/preferences_savefile.dm

```diff
@@ -207,6 +202,12 @@
 	S["adaptive_zoom"] >> adaptive_zoom
 	S["tooltips"] >> tooltips
 
+	S["key_bindings"] >> key_bindings
+	check_keybindings()
```

Comment:
Is this just for the save_prefs in check_keybindings? That doesn't look like a proper fix to me, it's still done before the sanitizing, should rather defer the saving